src/components/CouponDetails/index.jsx

```diff
@@ -658,7 +437,7 @@ class CouponDetails extends React.Component {
 
   isTableLoading() {
     const { couponDetailsTable } = this.props;
-    return couponDetailsTable && couponDetailsTable.loading;
+    return !!(couponDetailsTable && couponDetailsTable.loading);
```

Comment:
Ha, I thought only old school embedded C developers like me used this :P

Reply:
Guaranteed boolean all the time!